lib/milc_interface.cpp

```diff
@@ -878,14 +878,25 @@ void qudaInvert(int external_precision,
 
 #ifdef GPU_CLOVER_DIRAC
 
-void* qudaCreateExtendedGaugeField(void* gauge, int geometry, int precision)
+static inline void* createExtendedGaugeField(void* gauge, int geometry, int precision, int resident)
 {
   qudamilc_called<true>(__func__);
```

Comment:
Mathias, are we having the developers call? Did I miss something? Did I understand wrong the time???

> El 28/5/2015, a las 21:09, Mathias Wagner <notifications@github.com> escribió:
> 
> In lib/milc_interface.cpp <https://github.com/lattice/quda/pull/260#discussion_r31267213>:
> 
> > @@ -878,14 +878,25 @@ void qudaInvert(int external_precision,
> >  
> >  #ifdef GPU_CLOVER_DIRAC
> >  
> > -void* qudaCreateExtendedGaugeField(void* gauge, int geometry, int precision)
> > +static inline void* createExtendedGaugeField(void* gauge, int geometry, int precision, int resident)
> >  {
> >    qudamilc_called<true>(__func__);
> Just wondering whether we should us qudamilc_called<> only in exposed functions. 
> However, we can keep it this way for now and I will create a separate cleanup issue and we can decide whether we also want to have the qudamilc_called<false> call before exiting for all exposed functions.
> 
> But as this is only verbose output it does not prevent the merge. It just made me aware of another inconsistency in the milc interface.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/lattice/quda/pull/260/files#r31267213>.
>